applications/vila_live/README.md

```diff
@@ -0,0 +1,37 @@
+# Holoscan VILA Live
+
+This applications demonstrates how to run [VILA 1.5](https://github.com/Efficient-Large-Model/VILA) models on live video feed with the possibility
+of changing the prompt in real time.
```

Comment:
Could you please add 1-2 sentences summarizing what VILA is or does? What is its value proposition, or what distinguishes it from other HoloHub apps?

Reply:
Added